charts/splunk-connect-for-snmp/values.yaml

```diff
@@ -430,6 +434,14 @@ mongodb:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "9216"
+  pdb:
+    create: true
+
+  ## storageClass can be changed according to needs
+  persistence:
+    storageClass: "microk8s-hostpath"
```

Comment:
maybe let's add here that it should be changed if you use something else than microk8s (possibly with the docs/command of how to check what sotrageclasses you have?)